### PR TITLE
Fix arguments description

### PR DESCRIPTION
**Pull request by lvzii**
Fix arguments description, `output_max_length` was descripted as "the learning rate"

### CODE REVIEW

File: examples/stack_llama/scripts/rl_training.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ class ScriptArguments:
     reward_model_name: Optional[str] = field(default="", metadata={"help": "the model name"})
     log_with: Optional[str] = field(default=None, metadata={"help": "use 'wandb' to log with wandb"})
     learning_rate: Optional[float] = field(default=1.41e-5, metadata={"help": "the learning rate"})
-    output_max_length: Optional[int] = field(default=128, metadata={"help": "the learning rate"})
+    output_max_length: Optional[int] = field(default=128, metadata={"help": "maximum generating length"})
```
[the remark this answers is not in the document]
Comment (lvzii) [reply examples/stack_llama/scripts/rl_training.py]: Thanks for your suggestion, and I fixed another two description errors.

**Review by lvwerra — APPROVED** [review]
